protocol-designer/src/pages/Designer/ProtocolSteps/StepForm/StepFormToolbox.tsx

```diff
@@ -283,7 +288,16 @@ export function StepFormToolbox(props: StepFormToolboxProps): JSX.Element {
     dispatchAnalyticsEvent(FORM_WARNINGS_EVENT, visibleFormWarningsTypes)
     dispatchAnalyticsEvent(FORM_ERRORS_EVENT, visibleFormErrorsTypes)
   }, [visibleFormWarningsTypes, visibleFormErrorsTypes])
-
+  const moduleEntities = Object.entries(useSelector(getModuleEntities))
```

Comment:
you can simplify this a bit because you never need to reference the moduleId (which is the key in the [key,value] pair from doing Object.entries). You just need the length.

```suggestion
  const moduleEntities = Object.values(useSelector(getModuleEntities))
  const stackerModules = moduleEntities.filter((moduleEntity) => {
    return moduleEntity.type === FLEX_STACKER_MODULE_TYPE
  })
```